### PR TITLE
fix(generator-open-wc): add building-webpack generator

### PR DESCRIPTION
**Pull request by thepassle**

### CODE REVIEW

File: packages/generator-open-wc/generators/building-webpack/templates/static/index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!doctype html>
+<html>
+  <head></head>
+  <body>
+    <my-app></my-app>
```
Comment (thepassle) [inline packages/generator-open-wc/generators/building-webpack/templates/static/index.html]: Is there a way to make this less hardcoded? @daKmoR

File: packages/generator-open-wc/generators/building-webpack/templates/static/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import loadPolyfills from '@open-wc/polyfills-loader';
+
+loadPolyfills().then(() => {
+  import('./my-app.js');
```
Comment (thepassle) [inline packages/generator-open-wc/generators/building-webpack/templates/static/index.js]: Same question for this

File: packages/generator-open-wc/generators/building-webpack/templates/static/index.html (hunk 1 above)
Comment (daKmoR) [reply packages/generator-open-wc/generators/building-webpack/templates/static/index.html]: sure that would be a scaffold generator (take a look at the testing one)... so this also means that this should probably be split in two generator :p

**Comment by thepassle** [comment]
Not really sure how to split this up as two separate `scaffold-building` and `building-webpack`, for `building-webpack` you need an index.html and index.js, so then what would be the difference between `scaffold-building` and `building-webpack`? should we only make `scaffold-building` instead then?

**Comment by daKmoR** [comment]
`open-wc:building` runs
- `open-wc:building-webpack`
=> adopt package.json (adds scripts + dependencies)  
=> create webpack config file
e.g. modifies and creates new files (which you usally do not have in an existing project)

`open-wc:scaffold-building` runs
- `open-wc:building`
=> adds index.js
e.g. creates new files (which you usually have in an existing project)

File: packages/generator-open-wc/generators/building-webpack/templates/_package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open",
```
Comment (daKmoR) [inline packages/generator-open-wc/generators/building-webpack/templates/_package.json]: I think for your generators we will need to have a concept for all the scripts we will offer in the end
```
start => should be owc-dev-server? (in the end - not part of this generator)
start:build, start:building? => webpack-dev-server?
start:dist, start:built, start:constructed? => serving the dist (build vs built?)
```

I think I would go with `start:build`, `start:dist`  in this generator (+ rest stays as is)

Comment (thepassle) [reply packages/generator-open-wc/generators/building-webpack/templates/_package.json]: what about `"start:dev": "webpack-dev-server --mode development --open"` and `"start:prod": "http-server dist/ -o"`?

Comment (daKmoR) [reply packages/generator-open-wc/generators/building-webpack/templates/_package.json]: uh `start:prod` is nice 👍 

`start:dev` would suggest that just `start` is not for development...
`start:build` is sort of start the building server

but I have no super strong opinion about it...

Comment (thepassle) [reply packages/generator-open-wc/generators/building-webpack/templates/_package.json]: Lets go with this for now?
```
    "start:dev": "webpack-dev-server --mode development --open",
    "start:dev:es5": "webpack-dev-server --mode development --es5",
    "start:prod": "http-server dist/ -o",
```

**Pull request merged**